util/buildRelease/gen_release

```diff
@@ -248,7 +254,7 @@ foreach $dir (@code_dirs) {
         chomp $fullpath;
         $file = $fullpath;
         $file =~ s/(\S+\/)+//g;
-        if ($file =~ /(\.(h|cpp|c|ypp|lex)$)|Makefile|README|BUILD_VERSION/) {
+        if ($file =~ /(\.(h|cpp|c|ypp|lex|cmake)$)|Makefile|README|BUILD_VERSION|CMakeLists\.txt/) {
```

Comment:
I'm currently only seeing one `.cmake` file in the repo... do we expect there to be more over time?  Or do more get generated prior to this point in this script?  If the answers to both questions were "no", I'd probably explicitly name the single file in the previous @files list rather than using a glob pattern here.

Reply:
I think we can use the targeted approach. I was able to build from the tarball with `CHPL_LLVM=system` and `CHPL_LLVM=none` after limiting the copy to a specific file rather than the glob pattern.